src/common/exceptions/not-found.exception.ts

```diff
@@ -3,8 +3,13 @@ export class NotFoundException extends Error {
   readonly name: string = 'NotFoundException';
   readonly message: string;
 
-  constructor(path: string, readonly requestID: string) {
+  constructor(
+    path: string,
+    readonly requestID: string,
+    readonly err: string,
```

Comment:
Also, we should only get back `error` and `error_description` from 400 Bad Request errors, so we shouldn't add these properties to the other classes of errors.